flow/benchmarks/rllib/es_runner.py

```diff
@@ -0,0 +1,8 @@
+"""Runs the environments located in flow/benchmarks.
+
+The environment file can be modified in the imports to change the environment
+this runner script is executed on. This file runs the ES algorithm in rllib
+and utilizes the hyper-parameters specified in:
+
+TODO: add paper reference
```

Comment:
Evolution Strategies as a Scalable Alternative to Reinforcement Learning
by Salimans et. al.